test/system/src/test/resources/pravega.properties

```diff
@@ -22,7 +22,7 @@ bookkeeper.ack.quorum.size=3
 bookkeeper.write.timeout.milliseconds=10000
 bookkeeper.write.attempts.count.max=3
 controller.transaction.lease.count.max=600000
-controller.retention.frequency.minutes=2
+controller.retention.frequency.minutes=1
```

Comment:
Please make sure this change will not impact the existing test cases.

Reply:
I ran all the system tests with the retention frequency updated to 1 minute and they went through. System test 1 build number 4942